packages/dx-grid-core/src/plugins/virtual-table/helpers.ts

```diff
@@ -60,3 +66,26 @@ export const getViewport: GetViewportFn = (
 
   return result;
 };
+
+export const checkColumnWidths: TableColumnConverterFn = (tableColumns) => {
+  return tableColumns.reduce((acc, tableColumn) => {
+    const { width } = tableColumn;
+    if (typeof width === 'string') {
+      const numb = parseInt(width, 10);
+      const unit = numb ? width.substr(numb.toString().length) : width;
+      const isValidValue = VALID_UNITS.findIndex(validUnit => validUnit === unit) >= 0
+        && !isNaN(numb);
```

Comment:
The code above is the similar with code from `isValidValue` function

Reply:
`isValidValue` function placed in another plugin. We may try replace it in core.